### PR TITLE
Add unit test for B2BFeature

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Felt like increasing the coverage this Sunday evening. Add unit test for B2BFeature
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | PHPUnit test successes
| Possible impacts? | Nothing bad can happen.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26185)

### CODE REVIEW

File: tests/Unit/Adapter/B2b/B2bFeatureTest.php [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace Tests\Unit\Adapter\B2b;
```
Comment (Progi1984) [inline tests/Unit/Adapter/B2b/B2bFeatureTest.php]:
```suggestion
 */

declare(strict_types=1);

namespace Tests\Unit\Adapter\B2b;
```

File: tests/Unit/Adapter/B2b/B2bFeatureTest.php [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+use PHPUnit\Framework\TestCase;
+use PrestaShop\PrestaShop\Adapter\Configuration;
+use PrestaShop\PrestaShop\Core\B2b\B2bFeature;
+
+class B2bFeatureTest extends TestCase
+{
+    const CONFIGURATION_FEATURE = 'PS_B2B_ENABLE';
```
Comment (Progi1984) [inline tests/Unit/Adapter/B2b/B2bFeatureTest.php]:
```suggestion
    private const CONFIGURATION_FEATURE = 'PS_B2B_ENABLE';
```

**Review by Progi1984 — COMMENTED** [review]
@matks Thank you for your contribution. I have brought two comments but it's a good job.

**Review by sowbiba — APPROVED** [review]
Yaml linter not happy with files you didn't modify ^^

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @matks

**Pull request merged**